src/ng/directive/ngNonBindable.js

```diff
@@ -9,10 +9,13 @@
  *
  * @description
  * The `ngNonBindable` directive tells AngularJS not to compile or bind the contents of the current
- * DOM element (including the attributes of the element itself). This is useful if the element
- * contains what appears to be AngularJS directives and bindings but which should be ignored by
- * AngularJS. This could be the case if you have a site that displays snippets of code, for
- * instance.
+ * DOM element.AngularJS will generally ignore each attribute of the element itself. Attributes will
```

Comment:
I think the most succinct and accurate way of saying this is:

```
... tells AngularJS not to compile or bind the contents of the current DOM element, including attribute directives that have a lower priority than `ngNonBindable`. 
```

Reply:
Maybe `including attribute directives on the element (itself)` to make it clear that we are not referring to attribute directives on the children.
(Also, I don't think this is restricted to _attribute_ directives :grin:)

Reply:
so the final text would be:

> The `ngNonBindable` directive tells AngularJS not to compile or bind the contents of the
current DOM element, including directives on the element itself that have a
lower priority than `ngNonBindable`.
> This is useful if the element contains what appears to be AngularJS directives and bindings but which should be ignored by AngularJS. This could be the case if you have a site that displays snippets of code, for instance.

@jugglinmike would you make this change?

Reply:
Sure thing.